_includes/overall-schedule.md

```diff
@@ -23,5 +23,29 @@ Organizers will inform participants of the week schedule by email.
 {%- capture date %}{% if c.type != "Mentor-Mentee" %}{{ c.date }} {% if c.time %}([{{ c.time }} European Time](https://arewemeetingyet.com/Berlin/{{ c.date | date: "%Y-%m-%d" }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))){% endif %}{% endif %}{% endcapture %}
 | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: "(" | remove: ")" | remove: "@" | remove: ":" | remove: "," | replace: " ", "-" | remove: "&"  }}-calls) | {{ date }} | [**{{ c.title }}**](/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |
 {%- assign w-desc = "" %}
+{%- endfor %} 
 {%- endfor %}
-{%- endfor %}
+
+<script type="application/ld+json" >
+{
+  "@context": "https://schema.org",
+  "@id": "{{ site.url }}/{{ cohort }}/schedule",
+  "@type": "Course",
+  "dct:conformsTo": "https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08",
+  "description":  "{{site.description}}",
+    "hasCourseInstance": [
+    {
```

Comment:
The loop should be there. Not in `courseMode`

Reply:
I'm sorry this is not clear.

Reply:
Each cohort call in the different weeks should be a separate item in the `"hasCourseInstance"` list
So you need to move the loop over the weeks that you currently have in `courseMode`  there, and for each element, create the dictionary with all elements of a `courseInstance`
Is it more clear?

Reply:
Yes, thanks

Reply:
Good morning @bebatut tried to reach you via slack https://olsinternships.slack.com/archives/C04414774LE/p1666674279334609?thread_ts=1666187224.087659&cid=C04414774LE

Reply:
I moved the loop to the description as the loop gives an explicit definition of the event schedule. Does this work?